src/main/resources/templates/create.html

```diff
@@ -1,21 +1,45 @@
 <!DOCTYPE HTML>
 <html xmlns:th="https://www.thymeleaf.org">
-    <head>
-        <title th:text="|Edit survey ${survey.getTitle()}|">Edit survey</title>
-        <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
-        <link rel="stylesheet" type="text/css" media="all"
-              href="/css/style.css" th:href="@{/css/style.css}">
-    </head>
-    <body>
-        <p><a href="../home">Home</a></p>
-
-        <form method="post"
-              th:action="@{/create/{id}/update(id=${survey.getId()})}"
-              th:object="${survey}">
-            <!--Update survey title-->
-            <p>Title:<br><input th:field="*{title}" type="text"/></p>
-            <!--Save survey-->
-            <p><input type="submit" value="Save"/></p>
+<head>
+    <title th:text="|Edit survey ${survey.getTitle()}|">Edit survey</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" type="text/css" media="all" href="/css/style.css" th:href="@{/css/style.css}">
+</head>
+<body>
+<p><a href="../home">Home</a></p>
+
+<!-- Form to update survey title -->
+<form method="post" th:action="@{/create/{id}/update(id=${survey.getId()})}" th:object="${survey}">
+    <p>Title:<br><input th:field="*{title}" type="text"/></p>
+    <p><input type="submit" value="Save"/></p>
+</form>
+
+<!-- List of questions with edit and delete options -->
+<h3>Questions</h3>
+<ul>
+    <li th:each="question : ${survey.questions}">
+        <!-- Display question text -->
+
+
+        <!-- Edit form for each question -->
+        <form method="post" th:action="@{/create/{surveyID}/editQuestion/{questionID}(surveyID=${survey.getId()}, questionID=${question.id})}" style="display:inline;">
+            <input type="text" name="newQuestionText" placeholder="Edit question" th:value="${question.question}" />
+            <input type="submit" value="Update Question" />
```

Comment:
Could we change the value of the box to Save Question to be consistent with the Save button for the survey? It also makes it clearer because this button needs to be clicked for the question to be saved to the survey